.github/workflows/on-push-run-build.yml

```diff
@@ -0,0 +1,42 @@
+name: on-push-run-build
+on:
+  push:
+  
+env:
+  # Opt out of telemetry
+  DOTNET_CLI_TELEMETRY_OPTOUT: 1
```

Comment:
Consider adding `DOTNET_NOLOGO: 1` to avoid the spam in the console.

Reply:
I didn't know about this. I've included it. Thanks.